geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/InputFormatBaseAdapter.scala

```diff
@@ -0,0 +1,73 @@
+package org.locationtech.geomesa.jobs.mapred
+
+import org.apache.accumulo.core.client.mapred.AccumuloInputFormat
+import org.apache.accumulo.core.client.security.tokens.PasswordToken
+import org.apache.accumulo.core.security.Authorizations
+import org.apache.hadoop.mapred.JobConf
+import org.apache.hadoop.mapreduce.Job
+import org.apache.log4j.Level
+import org.locationtech.geomesa.accumulo.AccumuloVersion
+import AccumuloVersion._
+
+object InputFormatBaseAdapter {
+  def setConnectorInfo(job: JobConf, user: String, token: PasswordToken) = accumuloVersion match {
+    case V15 => setConnectorInfo15(job, user, token)
+    case V16 => setConnectorInfo16(job, user, token)
```

Comment:
I'm in favor of leaving in the duplicated methods.  It makes it very clear that they are using reflection and could possibly break on us.